ouster_client/include/ouster/client.h

```diff
@@ -97,6 +97,50 @@ std::shared_ptr<client> init_client(const std::string& hostname,
                                     timestamp_mode ts_mode = TIME_FROM_UNSPEC,
                                     int lidar_port = 0, int imu_port = 0,
                                     int timeout_sec = 60);
+
+/**
+ * Connect to sensor and start listening for data via multicast.
+ *
+ * @param[in] hostname hostname or ip of the sensor. 
+ * @param[in] config sensor config.
+ * @param[in] mtp_dest_host multicast ip address where the sensor should send data.
+ *
+ * @return pointer owning the resources associated with the connection.
+ */
+std::shared_ptr<client> mtp_init_client(const std::string& hostname,
+                                        const sensor_config& config,
+                                        const std::string& mtp_dest_host);
+
+/**
+ * Connect to and configure the sensor and start listening for data via multicast.
+ *
+ * @param[in] hostname hostname or ip of the sensor. 
+ * @param[in] config sensor config to set on sensor.
+ * @param[in] mtp_dest_host multicast ip address where the sensor should send data.
+ * @param[in] timeout_sec how long to wait for the sensor to initialize.
+ *
+ * @return pointer owning the resources associated with the connection.
+ */
+std::shared_ptr<client> mtp_init_client_main(const std::string& hostname,                                    
+                                             const sensor_config& config,
+                                             const std::string& mtp_dest_host = "",
+                                             int timeout_sec = 60);
+
+
+/**
+ * Listen for sensor data on the specified ports via multicast; do not configure the sensor.
+ *
+ * @param[in] hostname hostname or ip of the sensor. 
+ * @param[in] config active sensor config from sensor.
+ * @param[in] mtp_dest_host multicast ip address where the sensor should send data.
+ * @param[in] timeout_sec how long to wait for the sensor to initialize.
+ *
+ * @return pointer owning the resources associated with the connection.
+ */
+std::shared_ptr<client> mtp_init_client_slave(const std::string& hostname,                                    
```

Comment:
Can you rename to:
mtp_init_client_secondary

Reply:
Done in [77650a5](https://github.com/ouster-lidar/ouster_example/pull/480/commits/77650a5c611b0657ab9b5247802f2d20bc89fbd7)